fc/build-from-security.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python3
+
+import apt
+import atexit
+import glob
+import os
+import shutil
+import subprocess
+import sys
+import tempfile
+
+arch_to_base_uri = {
+    "amd64": "http://archive.ubuntu.com/ubuntu",
+    "i386":  "http://archive.ubuntu.com/ubuntu",
+    # ports
+    "s390x": "http://ports.ubuntu.com/ubuntu-ports",
+    "arm64": "http://ports.ubuntu.com/ubuntu-ports",
+    "armhf": "http://ports.ubuntu.com/ubuntu-ports",
+    "ppc64el": "http://ports.ubuntu.com/ubuntu-ports",
+}
+
+def fetch_source_from_security(release, pkgname):
+    apt.apt_pkg.config.set("Acquire::AllowInsecureRepositories", "1")
```

Comment:
Why? Perhaps add a comment?

Reply:
Good point, to make this work I need to setup more things in this mini apt environment, but I can try to see if that is possible without too much hassle. I.e. I need a ubuntu-keyring in my fake aptroot.

Reply:
+1 from me once you address AllowInsecureRepositories.